specification/hybridcloud/resource-manager/Microsoft.HybridCloud/preview/2022-11-01-preview/hybridcloud.json

```diff
@@ -0,0 +1,992 @@
+        },
+        "remoteResourceId": {
+          "type": "string",
+          "description": "Identifier for the remote resource"
+        },
+        "sharedKey": {
+          "type": "string",
+          "description": "Shared Key of the VPN Link Connection"
+        },
+        "provisioningState": {
+          "readOnly": true,
+          "$ref": "#/definitions/ProvisioningState",
+          "description": "The provisioning state of the application rule collection resource."
+        }
+      },
+      "description": "Cloud Connector resource properties."
+    },
+    "CloudConnection": {
+      "type": "object",
+      "description": "Resource which represents the managed network connection between Azure Gateways and remote cloud gateways",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/CloudConnectionPropertiesFormat",
+          "description": "Properties of the Cloud Connection."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/TrackedResource"
+        }
+      ]
+    },
+    "DiscoverResources": {
+      "type": "object",
+      "description": "Resource that represents resources found in remote cloud environments",
+      "properties": {
+        "resourceFilter": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ResourceFilter",
+          "description": "Remote Cloud Resource Type filter"
+        },
+        "discoveredResources": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DiscoveredResource"
+          },
+          "x-ms-identifiers": [
+            "id"
+          ],
+          "description": "A list of discovered remote cloud resources."
+        }
+      }
+    },
+    "ResourceFilter": {
+      "type": "object",
+      "description": "Remote Cloud Resource Type filter",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Resource name.",
+          "readOnly": true
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource type."
+        },
+        "location": {
+          "type": "string",
+          "description": "Resource location."
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Resource tags."
+        }
+      }
+    },
+    "DiscoveredResource": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Remote cloud resource identifier",
+          "readOnly": true
+        },
+        "body": {
+          "type": "string",
+          "description": "Details / Properties of the remote cloud resource.",
+          "readOnly": true
+        }
```

Comment:
If it can be XML or JSON, you can add an enum property indicating that, somehting like, ContentType or something. How will client know how to parse the string, as XML or JSON?

Reply:
Added